### PR TITLE
100/errors and cleanup

### PR DESCRIPTION
**Pull request by JDziewonski98**
# Summary

Cleaned up service and controller for collection, fixed tests, same old

## Test Plan

Ran updated unit tests

## Related Issues
100 WIP

### CODE REVIEW

File: backend/src/controllers/muralcollection.controller.ts [hunk 1]
```diff
@@ -10,18 +10,23 @@ export class MuralCollectionController {
   public collectionService: MuralCollectionService = new MuralCollectionService();
 
   // POST /collection
+  /**
+   * POST /collection
+   * @param req HTTP request containing a "collection" parameter that has a
+   * MuralCollectionInterface describing the collection, and a list of mural IDs
+   * @param res Http response
```
Comment (tedspare) [inline backend/src/controllers/muralcollection.controller.ts:17]: have you been describing the responses as well?

File: backend/src/controllers/tour.controller.ts [hunk 2]
```diff
@@ -42,7 +42,12 @@ export class TourController {
     }
   }
 
-  // PUT update a mural with a given id
+  /**
+   * PUT /tour update a tour with a given id
+   * @param req HTTP request containing a tourId and a tourInterface
+   *  describing the new tour to create
+   * @param res HTTP response
```
Comment (tedspare) [inline backend/src/controllers/tour.controller.ts:49]: same point here

File: backend/src/services/muralcollection.service.ts [hunk 3]
```diff
@@ -28,7 +47,20 @@ export class MuralCollectionService {
...
+            {
+              model: Mural,
+              as: "murals",
+              attributes: ["id"],
+              through: {
+                attributes: [],
+              },
+            },
+          ],
+          attributes: { exclude: ["updatedAt", "createdAt"] },
```
Comment (tedspare) [inline backend/src/services/muralcollection.service.ts:62]: nice addition

File: backend/src/services/tests/muralcollection.test.ts [hunk 4]
```diff
@@ -57,12 +58,13 @@ test("show valid mural collection", async () => {
   const collectionService: MuralCollectionService = new MuralCollectionService();
 
   const murals: number[] = [1, 2];
-  const create = await collectionService.create(params, murals);
-  if (!create.success) {
-    fail("Creating mural collection failed.");
-  }
+  const create = await collectionService
+    .create(params, murals)
+    .catch((err: Error) => fail());
```
Comment (tedspare) [inline backend/src/services/tests/muralcollection.test.ts:63]: a bit late but should these `fail`s have messages like before?

**Review by tedspare — APPROVED** [review]
LGTM to merge - a few questions but nothing critical

File: backend/src/services/tests/muralcollection.test.ts (hunk 4 above)
Comment (JDziewonski98) [reply backend/src/services/tests/muralcollection.test.ts:63]: Yeah it's better if they do - there are so many though I'm kind of ignoring them rn

File: backend/src/controllers/muralcollection.controller.ts (hunk 1 above)
Comment (JDziewonski98) [reply backend/src/controllers/muralcollection.controller.ts:22]: Yeah I should, will probably do that in the next pass lol

**Pull request merged**